dev/TEAL_opcodes.md

```diff
@@ -1365,3 +1387,33 @@ The notation A,B indicates that A and B are interpreted as a uint128 value, with
 - Ath value of the array field F from the Tth transaction in the last inner group submitted
 - Availability: v6
 - Mode: Application
+
+## vrf_verify s
+
+- Opcode: 0xd0 {uint8 parameters index}
+- Stack: ..., A: []byte, B: []byte, C: []byte &rarr; ..., X: []byte, Y: uint64
+- Verify the proof B of message A against pubkey C. Returns vrf output and verification flag.
+- **Cost**: 5700
+- Availability: v7
+
+`vrf_verify` Standards:
+
+| Index | Name | Notes |
+| - | ------ | --------- |
+| 0 | VrfAlgorand |  |
+
```

Comment:
The official name is:
ECVRF-EDWARDS25519-SHA512-Elligator2 from https://tools.ietf.org/id/draft-irtf-cfrg-vrf-06.html

Reply:
We don't want to make that the constant name, do we?

Also, I thought someone said that the ietf draft ended up being slightly different somehow?